volatility3/framework/symbols/linux/extensions/__init__.py

```diff
@@ -578,7 +578,7 @@ def get_name(self, context, task):
         return fname
 
     # used by malfind
-    def is_suspicious(self):
+    def is_suspicious(self, proclayer):
```

Comment:
This will be an api change, unless you default it to None and then verify whether `proclayer` has been set before using it...

Reply:
Ok, I've done that.